.github/workflows/accept.yml

```diff
@@ -0,0 +1,186 @@
+name: Fallback Install, Build & Package & Prebuilt Install (Merge)
+
+on: 
+  push: 
+    branches: 
+      # triggered by pull request approval triggers which is a merge to default branch
+      - master
+
+  workflow_dispatch:
+    inputs: 
+      target-test:
+        required: false
+        description: 'Run Tests on Target Group? (type: yes)'
+        default: false
+
+jobs:
+  # entry point 1: fallback group
+  load-fallback-group:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout ${{ github.ref }}
+      uses: actions/checkout@v2
+
+      # *** comments for: fallback-group.json
+      # contains images that are able to build "out of the box"
+    - name: Load target group data
+      id: set-matrix
+      run: .github/scripts/matrix-from-json.sh .github/config/fallback-group.json
+
+  fallback-group-install:
+    runs-on: ubuntu-latest
+    needs: load-fallback-group
+    strategy:
+      fail-fast: false
+      matrix: ${{fromJson(needs.load-fallback-group.outputs.matrix)}}
+
+    env:
+      AO_TOKEN_PROD: ${{ secrets.AO_TOKEN_PROD }}
+
+    steps:
+      - name: Checkout ${{ github.ref }}
+        uses: actions/checkout@v2
+
+      - name: Create Container ${{ matrix.image }}
+        id: create
+        run: .github/scripts/container-from-matrix-input.sh ${{ matrix.image }}
+
+      # the current version package has yet to be built.
+      # it will fallback to build from source
+      - name: NPM Install (to fallback)
+        run: docker exec ${{ steps.create.outputs.containerId }} npm install --unsafe-perm
```

Comment:
Can the output from the npm install command or some compilation artifact be checked, to help verify (other than clicking through all the step output for each job) that "build from source" is actually happened?

Reply:
Generally if the install did not work - the test will fail.

That said, there was a Show Artifacts step in push/pull triggered workflows. So following that spirit I added a Confirm Install Artifacts step that explicitly checks installed/build artifacts are indeed in dist directory. Similar is used in all workflows.

Implementation 'hard codes' napi7. Note that current workflows do not check napi4 explicitly (all node 10 versions are napi7) but do build the napi4 tarballs.